Re-organize config_pes.xml file

TomasTorsvik opened the pull request:
Add options for 1-node configurations (128 CPUs) on Betzy.

TomasTorsvik committed:
Re-organize config_pes.xml file

JorgSchwinger commented on cime_config/config_pes.xml:
```diff
@@ -1,81 +1,129 @@
 
+  <!-- Configurations for the one degree tripolar (tnx1) grid  !-->
+
   <grid name="a%T62.+oi%tnx1v1">
```
I am wondering if we need to support the tnx1v1 and tnx1v3 grids  any longer. As far as I know, they were never used for any production runs(?) @matsbn might have an opinion about this.

TomasTorsvik replied:
I have never used tnx1v1 and tnx1v3. I did not remove any existing definitions from the file, but I can do it if we all agree.

JorgSchwinger commented on cime_config/config_pes.xml:
```diff
@@ -478,203 +620,166 @@
-	  <rootpe_wav>1</rootpe_wav> 
-	  <rootpe_cpl>0</rootpe_cpl>                         
-	</rootpe>
+      <pes pesize="S" compset="_DATM.*_BLOM%ECO">
```
This is confusing: There is on pesize="S" layout with 155 ocean tasks, one pesize="M" layout with 155 ocean tasks further below, and one pesize="M" layout with 272 ocean tasks. It is a bit difficult to see what is new and what has been there before.

TomasTorsvik replied:
These are old pe-layouts, I only added new `XS` layouts.
I agree that it is confusing, but I didn't want to remove something that someone might still be using.

JorgSchwinger reviewed (commented):
Thank you Tomas. I am not sure how far we want to maintain "old" pe-layouts. In my opinion, on the way forward to NorESM2.5 there is not really the need to keep everything.

TomasTorsvik commented:
At the moment I have kept all the old definitions, just moved some things around, and added two `XS` definitions for `T62_tn14` and `TL319_tn14`. I agree that we could simplify the file, as we are now mostly running on Betzy, where the relevant pe-layouts are 128 CPUs for the "devel" queue and 512 or larger for the "normal" queue.

TomasTorsvik commented:
@JorgSchwinger @jmaerz @matsbn @milicak 
Hi, if there are no objections, I would like to merge this as it makes possible running short tests on the "devel" queue on Betzy.

jmaerz reviewed (approved):
Hi @TomasTorsvik , I can't comment much on it from my side. So from my side no objections.
